suite-native/module-send/src/screens/SendFeesScreen.tsx

```diff
@@ -16,6 +18,24 @@ export const SendFeesScreen = ({
         selectAccountByKey(state, accountKey),
     );
 
+    useEffect(() => {
+        // Subscribe to blocks for Solana, since they are not fetched globally
+        // this is needed for correct Solana fee estimation
+        if (account && account.networkType === 'solana') {
+            TrezorConnect.blockchainSubscribe({
```

Comment:
nit: maybe create variable for that params and use in both calls?

Reply:
This is a copy of what @martykan used for desktop.

Reply:
That doesn't mean it coudn't be improved a bit :D